projects/addon-doc/src/interfaces/code-editor.ts

```diff
@@ -1,5 +1,15 @@
-export interface CodeEditor {
+export interface TuiCodeEditor {
     readonly name: string;
+    edit(component: string, id: string, files: Record<string, string>): Promise<void>;
 
+    /**
+     * @deprecated
+     * use `edit` method instead of `open`
+     */
     open(component: string, sampleId: string, files: Record<string, string>): void;
 }
+
+/**
+ * @deprecated use {@link TuiCodeEditor}
+ */
+export type CodeEditor = TuiCodeEditor;
```

Comment:
can you add this to migration please? https://github.com/Tinkoff/taiga-ui/blob/main/projects/cdk/schematics/ng-update/constants/types.ts
`from CodeEditor, to TuiCodeEditor, moduleSpecifier @taiga-iu/addon-doc`

Reply:
fixed